frontend/src/bundles/studio/pages/studio.tsx

```diff
@@ -39,8 +48,40 @@ const Studio: React.FC = () => {
     }, [dispatch]);
 
     const handleSubmit = useCallback(() => {
-        navigate(AppRoute.ROOT);
-    }, [navigate]);
+        // TODO: REPLACE LOGIC WITH MULTIPLE SCENES
+        const scene = scenes[0];
+        const script = scripts[0];
+
+        if (!scene?.avatar && !script) {
+            return;
```

Comment:
Maybe we can show toast notification or modal that to create video at least one scene and script must be added?